source/_components/switch.edimax.markdown

```diff
@@ -24,10 +24,23 @@ switch:
     host: 192.168.1.32
 ```
 
-Configuration variables:
-
-- **host** (*Required*): The IP address of your Edimax switch, eg. `192.168.1.32`.
-- **username** (*Optional*): Your username for the Edimax switch. Defaults to `admin`.
-- **password** (*Optional*): Your password for the Edimax switch. Defaults to `1234`.
-- **name** (*Optional*): The name to use when displaying this switch.
-
+{% configuration %}
+host:
+  description: "The IP address of your Edimax switch, e.g., `192.168.1.32`."
+  required: true
+  type: string
+username:
+  description: Your username for the Edimax switch.
+  required: false
+  default: admin
+  type: string
+password:
+  description: Your password for the Edimax switch.
+  required: false
+  default: 1234
+  type: string
+name:
```

Comment:
Feel free to check the documentation with the actual component code.
In this case the name defaults to 'Edimax Smart Plug', 
https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/switch/edimax.py#L20

Reply:
oops I forgot, I always have the code to check 😉